### PR TITLE
Updates HG Analysis to include GDCC

### PR DESCRIPTION
**Pull request by amezcum1**
Updates host-guest analysis to include GDCC (TEMOA/TEETOA) dataset. Also added reference calculations as non-ranked submissions, and includes analysis of GDCC without optional TEMOA-G3 system (GDCC_no_optional directories). 

When you get a chance can you have a look at the **_experimental_measurements.pdf_** table in  **_/SAMPL8/host_guest/Analysis/ExperimentalMeasurements/PDFTable/ directory_** ? Particularly the notation of the measured Ka values, and the location of the superscripts of (next to host-guest system names) in the table. Is the table easy to follow? Informative enough? 

Thank you in advance.

### CODE REVIEW

**Comment by davidlmobley** [comment]
On the experimental table -- I'd say this works for a first pass. There are some minor details I'd probably want to clean up in a revision
- NaN usually means infinity; probably better to make these "ND" (not detected) or similar
- The large values would be much easier to work with if in scientific notation, e.g. for PCP I'd get stuck with counting zeros to figure out what those numbers mean.
- The plus or minus "+-" could be formatted as latex $\pm$ which gets an actual "plus or minus" symbol

The footnotes/subscripts seem fine to me as is.

Leaving this here for now and will try and take a quick pass through the rest, or at least as much as I have time for.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ We will make available challenge instructions for the logD challenge, as well as
 - 2021-04-13: Add GSK challenge deadline
 - 2021-05-04: Add a submission template and an example submission file for both the pKa and logD challenge. Add enumerated microstates for the challenge molecules.
 - 2021-07-05: Update example submission for pKa challenge; add pKa challenge instructions; update README files.
+- Add GDCC (TEMOA/TEETOA) to SAMPL8 host-guest analysis, add/update preliminary statistics and analysis tables, plots, and figures for GDCC dataset. (7/6/21)
```
Comment (davidlmobley) [inline README.md:60]: Can you put a date for this one, please?

File: host_guest/Analysis/All_Accuracy/README.md [hunk 2]
```diff
@@ -3,5 +3,7 @@
 ## Manifest
 
 - `CB8`(under construction): Analysis and results of all methods for CB8 systems
+- `GDCC`(under construction): Analysis and results of all methods for GDCC (TEMOA/TEETOA) systems
```
Comment (davidlmobley) [inline host_guest/Analysis/All_Accuracy/README.md:6]: Should these still say "under construction"?

File: host_guest/Analysis/Ranked_Accuracy/README.md [hunk 3]
```diff
@@ -3,5 +3,7 @@
 ## Manifest
 
 - `CB8`(under construction): Analysis and results of ranked methods for CB8 systems.
+- `GDCC`(under construction): Analysis and results of ranked methods for GDCC (TEMOA/TEETOA) systems.
```
Comment (davidlmobley) [inline host_guest/Analysis/Ranked_Accuracy/README.md:6]: Still "under construction"?

**Review by davidlmobley — APPROVED** [review]
Looks good to merge once you make a few minor fixes to docs. Thanks for all your work on this!

File: README.md (hunk 1 above)
Comment (amezcum1) [reply README.md]: added date in the same format as entries prior.

File: host_guest/Analysis/Ranked_Accuracy/README.md (hunk 3 above)
Comment (amezcum1) [reply host_guest/Analysis/Ranked_Accuracy/README.md]: don't need it there.

**Pull request merged**